Dockerfile

```diff
@@ -37,9 +37,10 @@ RUN \
   tar xf \
     /tmp/kimai.tar.gz -C \
     /app/www --strip-components=1 && \
-  rm -rf /app/www/var && \
```

Comment:
note for self: i was wondering why remove this but i see we're doing it during the init as well https://github.com/linuxserver/docker-kimai/blob/88b5c29d964cbe80e5a710e5cde171fb24ad8c23/root/etc/s6-overlay/s6-rc.d/init-kimai-config/run#L29 to make room for symlinks to `/config`